package.json

```diff
@@ -29,20 +29,25 @@
     "lint:php": "composer lint",
     "format": "npm-run-all --parallel format:*",
     "format:php": "composer format",
+    "format:js": "node_modules/.bin/prettier --write js/src/*.js",
```

Comment:
This should actually be `"format:js": "wp-scripts format-js ./js/src"`, the use of prettier is not needed.

Reply:
I did not know about  `wp-scripts format-js ./js/src`. However it seems it still use prettier which needs to be installed as a dependency because when i tried to run that command, it say that prettier cannot be found. So I tried to install the normal version of prettier (not wp-prettier as the command 'wp-scripts format-js ./js/src' can't use it) and it works, except that it does not take the `parenSpacing` option (spaces before and after parenthesis) that only `wp-prettier` bring.
Any thoughts?

Reply:
I added just this `"format:js": "wp-scripts format-js js/src"` to the `package.json` and added the `.prettierrc.json` and ran the command and it worked perfectly for me.